src/local-development/delete-local-component.ts

```diff
@@ -0,0 +1,35 @@
+import { AppComponentType } from '../types/app-component-type.types';
+import { log } from '../output-channel';
+import { removeModuleFromGroups } from './groups-json';
+import vscode from 'vscode';
```

Comment:
Import everywhere else is "* as":

```suggestion
import * as vscode from 'vscode';
```